src/main/java/kitchenpos/order/application/OrderService.java

```diff
@@ -0,0 +1,94 @@
+package kitchenpos.order.application;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import kitchenpos.menu.domain.Menu;
+import kitchenpos.menu.domain.MenuRepository;
+import kitchenpos.order.domain.Order;
+import kitchenpos.order.domain.OrderLineItem;
+import kitchenpos.order.domain.OrderLineItemRepository;
+import kitchenpos.order.domain.OrderMenu;
+import kitchenpos.order.domain.OrderRepository;
+import kitchenpos.order.domain.OrderStatus;
+import kitchenpos.order.dto.OrderRequest;
+import kitchenpos.order.dto.OrderResponse;
+import kitchenpos.order.dto.OrderStatusRequest;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional(readOnly = true)
+@Service
+public class OrderService {
+
+    private final MenuRepository menuRepository;
+    private final OrderRepository orderRepository;
+    private final OrderLineItemRepository orderLineItemRepository;
+
+    public OrderService(final MenuRepository menuRepository,
+                        final OrderRepository orderRepository,
+                        final OrderLineItemRepository orderLineItemRepository) {
+        this.menuRepository = menuRepository;
+        this.orderRepository = orderRepository;
+        this.orderLineItemRepository = orderLineItemRepository;
+    }
+
+    @Transactional
+    public OrderResponse create(final OrderRequest orderRequest) {
+        Order order = new Order(orderRequest.getOrderTableId());
+
+        Order savedOrder = orderRepository.save(order);
+        List<OrderLineItem> orderLineItems = orderLineItemRepository.saveAll(
+            newOrderLineItems(savedOrder, orderRequest)
+        );
+
+        return OrderResponse.from(savedOrder, orderLineItems);
+    }
+
+    private List<OrderLineItem> newOrderLineItems(final Order order,
+                                                  final OrderRequest orderRequest) {
+        if (Objects.isNull(orderRequest.getOrderLineItems())) {
+            throw new IllegalArgumentException();
+        }
+
+        final Map<Long, Long> menuIdAndQuantity = orderRequest.toMap();
```

Comment:
```suggestion
        Map<Long, Long> menuIdAndQuantity = orderRequest.toMap();
```

갑자기 final이 껴있으니 헷갈리네용. 통일해주면 좋을 것 같아서요 :)

Reply:
에고 실수했네요 🥲